clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -544,6 +544,31 @@ describe("hubConnection", () => {
         }
     });
 
+    it("transport falls back from WebSockets to SSE", async (done) => {
+        // Replace Websockets with a function that just
+        // throws to force fallback.
+        const oldWebSocket = (window as any).WebSocket;
+        (window as any).WebSocket = () => {
+            throw new Error("Kick rocks");
+        };
+
+        const hubConnection = new HubConnection(TESTHUBENDPOINT_URL, {
+            logger: LogLevel.Trace,
+            protocol: new JsonHubProtocol(),
+        });
+
+        try {
+            await hubConnection.start();
+
+            // Make sure that we connect with SSE.
```

Comment:
This won't be true on IE when we test there. Let's check that it's either ServerSentEvents OR LongPolling.

Reply:
Good catch 👍